ideascube/serveradmin/catalog.py

```diff
@@ -306,6 +317,88 @@ def remove(self, install_dir):
             printerr(e)
 
 
+class StaticSite(SimpleZipPackage):
+    typename = 'static-site'
+    handler = Nginx
+
+
+class ZippedMedia(SimpleZipPackage):
```

Comment:
Nope, "media" is already the plural form.

https://en.wiktionary.org/wiki/media#Etymology_2